docker-compose.yaml

```diff
@@ -0,0 +1,12 @@
+version: '3'
+
+services:
+
+  postgre:
```

Comment:
Nitpicking: 'postgre' intentionally ?

Reply:
Good point. What is the right name?
I checked the official page before writing that: https://www.postgresql.org/
What do you suggest to name it? Maybe although the real name is `postgre` people call it something else?

Reply:
It is absolutely fine if you purposefully wrote it. Kindly ignore my comment.